package.json

```
@@ -8,8 +8,14 @@
   "author": "Kontist GmbH",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "tsc && cp -r src/templates dist/src",
+    "clean:dist": "rm -rf dist && mkdir -p dist/src",
+    "copy:assets": "cp -r src/templates dist/src",
+    "build": "npm run clean:dist && npm run copy:assets && tsc",
+    "build:hot": "npm run clean:dist && npm run copy:assets && tsc --watch",
+    "build:docker": "docker build -f Dockerfile.dev -t mocksolaris .",
     "start": "node dist/src/server.js",
+    "start:hot": "nodemon --watch dist/ --exec npm run start",
+    "start:docker": "docker run -p 127.0.0.1:2091:2091 -v $(pwd)/dist:/opt/mockSolaris/dist/ mocksolaris npm run start:hot",
```

Let's add the templates here as well to allow a hot reload:
```suggestion
    "start:docker": "docker run -p 127.0.0.1:2091:2091 -v $(pwd)/dist:/opt/mockSolaris/dist/ -v $(pwd)/src/templates:/opt/mockSolaris/dist/src/templates/ mocksolaris npm run start:hot",

```